### PR TITLE
Apply Checkstyle, Errorprone etc. for all Java modules (FINERACT-1171)

### PR DESCRIPTION
**Pull request by vorburger**
This is a subset (extract) from #1416

Attribution to Aleks is preserved!

for FINERACT-1171

### CODE REVIEW

**Comment by vorburger** [comment]
@vidakovic as per comment in #1416, how about we already do (just) this, and hold off with the `fineract-api` a little? Would you LGTM this? Note how while this PR appears to be from me, which doesn't really matter, the commit itself is still attributed (authored) by you!

@ptuomola do you want to review and LGTM this?

**Comment by vorburger** [comment]
FTR: I've stared at this a bit, and it's just moves (I've actually verified we don't "loose" anything) - so LGTM from me on this!

**Review by vidakovic — APPROVED** [review]
LGTM

**Pull request merged**
